src/freenet/clients/http/wizardsteps/DATASTORE_SIZE.java

```diff
@@ -44,11 +44,19 @@ public void getStep(HTTPRequest request, PageHelper helper) {
 
 		@SuppressWarnings("unchecked")
 		Option<Long> sizeOption = (Option<Long>) config.get("node").getOption("storeSize");
+		@SuppressWarnings("unchecked")
+		Option<Long> clientCacheSizeOption = (Option<Long>) config.get("node").getOption("clientCacheSize");
+		@SuppressWarnings("unchecked")
+		Option<Long> slashdotCacheSizeOption = (Option<Long>) config.get("node").getOption("slashdotCacheSize");
 		if(!sizeOption.isDefault()) {
 			long current = sizeOption.getValue();
+			long currentTotal = current;
+			if (!clientCacheSizeOption.isDefault() && !slashdotCacheSizeOption.isDefault()) {
+				currentTotal += clientCacheSizeOption.getValue() + slashdotCacheSizeOption.getValue();
+			}
```

Comment:
Does this mean that the clientCacheSizeOption value is only added if it is not the default?

Is the logic correct if it *is* the default?

Reply:
Yeah I wasn't really sure what to do if those are default...
And we only arrive here if `!sizeOption.isDefault()`, so I don't know if they can even be default at this point 🤔

Perhaps it would make more sense to just put this as a replacement of line 52:
`long current = sizeOption.getValue() + clientCacheSizeOption.getValue() + slashdotCacheSizeOption.getValue();`

Reply:
After thinking it over, I do think it makes more sense to do it this way 9fa12bbf63fb6106738ebb4addb46dceb4a58568